cmd/renderer.go

```diff
@@ -45,6 +46,8 @@ func (rt RendererTable) Render(v interface{}) error {
 		rt.renderJobs(*typed)
 	case *presenters.JobSpec:
 		rt.renderJob(*typed)
+	case *models.BridgeType:
+		rt.renderBridge(*typed)
```

Comment:
I'm not sure I fully understand this. Does that mean we would have to define a string with the expected output from `Render` against which we would then assert the content of what is written to `testWriter` ?

Reply:
We can punt on this idea for now and tackle it in another PR if you prefer.

Reply:
type testWriter struct {
    	expected string
    	t        testing.TB
    	found    bool
    }
    
    func (w *testWriter) Write(actual []byte) (int, error) {
    	fmt.Println(bytes.Index(actual, []byte(w.expected)))
    	if bytes.Index(actual, []byte(w.expected)) != -1 {
    		w.found = true
    	}
    	return len(actual), nil
    }
    
    func TestRendererTableRenderBridge(t *testing.T) {
    	bridge := models.BridgeType{Name: "hapax", 
                                        URL: cltest.WebURL("http://hap.ax"), 
                                        DefaultConfirmations: 0}
    	tw := &testWriter{bridge.Name, t, false}
    	r := cmd.RendererTable{Writer: tw}
    	assert.Nil(t, r.Render(&bridge))
    	assert.Equal(t, tw.found, true)
    }

This is what I've came up with so far, took the assert out of the `testWriter` since `Render` outputs several different lines, but it seems a bit sloppy.

Reply:
I think this is good as is for now. Better than what we have now. Put it in and I will go ahead and merge.